keepassxc-browser/background/keepass.js

```diff
@@ -945,25 +945,18 @@ keepass.updateDatabaseHashToContent = async function() {
     }
 };
 
-keepass.compareVersion = function(minimum, current, canBeEqual = true) {
-    if (!minimum || !current) {
-        return false;
-    }
-
-    // Handle beta/snapshot builds as stable version
-    const snapshot = '-snapshot';
-    const beta = '-beta';
-    if (current.endsWith(snapshot)) {
-        current = current.slice(0, -snapshot.length);
+// Expects an array of versions to compare
+keepass.compareVersions = function(versions, current, canBeEqual = true) {
```

Comment:
compareVersion (the global function) and keepass.compareVersions (the class function) are way too similar to each other which makes it hard to read and debug which function you actually intend to use. Recommend different naming choices.